wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/sysid/subsystems/Drive.java

```diff
@@ -122,10 +122,20 @@ public Command arcadeDriveCommand(DoubleSupplier fwd, DoubleSupplier rot) {
         .withName("arcadeDrive");
   }
 
+  /**
+   * Returns a command that will execute a quasistatic test in the given direction.
```

Comment:
Can this have a blurb about what a quasistatic test does, a la https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/creating-routine.html#types-of-tests